app/src/test/java/com/kickstarter/libs/LakeTest.kt

```diff
@@ -44,12 +44,32 @@ class LakeTest : KSRobolectricTestCase() {
 
     private fun assertDefaultProperties(user: User?) {
```

Comment:
Should we consider renaming these tests to `assertSessionProperties` so that it's consistent with the `sessionProperties` function name?

Reply:
so these will include all properties including the session, context, and user groups so i don't think `default` is right but not sure `session` is true either. maybe just `assertProperties`?

Reply:
Ahh gotcha. I think maybe default makes sense then.

Reply:
so i renamed this method but left the test methods as `assertDefaultProperties`!